router/router.nginx.conf.in

```diff
@@ -111,18 +111,14 @@ server {
     }
 
     location @pod_dead {
-        return 307 $updated_scheme://$updated_host/new;
-    }
-
-    location ~ /instance-ready/([^/]+)/ {
-        resolver kube-dns.kube-system.svc.cluster.local;
-        proxy_pass http://$1.default.svc.cluster.local/instance/$1/;
+        rewrite ^ @notebook_base_path@/error;
+    	proxy_pass http://notebook;
     }
 
     location / {
-        proxy_pass http://notebook/;
+        proxy_pass http://notebook;
 
-        proxy_set_header Host $updated_host;
+        # proxy_set_header Host $updated_host;
```

Comment:
Why do we not want the host? I don't understand the router well enough.

Reply:
Good catch, I meant to delete these.

We do not want the updated host: that's the host from the original request, which in the case of external use of dev namespaces, is `internal.hail.is`.  Now that batch/workshop is dispatching off the Host field (see aiohttp Application.add_domain), the host needs to be batch(.stuff) or workshop(.stuff) for things to route properly.